.github/workflows/neofs.yml

```diff
@@ -0,0 +1,29 @@
+name: Publish to NeoFS
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  push-to-neofs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set PATH_TO_FILES_DIR
+        run: |
+          echo "PATH_TO_FILES_DIR=output" >> $GITHUB_ENV
```

Comment:
why is it hardcoded and not an env? @vvarg229

Reply:
> why is it hardcoded and not an env? @vvarg229

Yes, you can put this variable into GitHub variables from the web interface

Reply:
Looks like there is some inconsistency there, we have `PATH_TO_FILES_DIR` in an `env`, but here it's hardcoded to a particular value appropriate for `make docker/pdf`.